src/widgets/dialogs/EmotePopup.hpp

```diff
@@ -29,8 +30,19 @@ class EmotePopup : public BasePopup
     ChannelView *channelEmotesView_{};
     ChannelView *subEmotesView_{};
     ChannelView *viewEmojis_{};
+    ChannelView *searchView_{};
```

Comment:
```suggestion
    /**
     * @brief Visible only when the user has specified a search query into the `search_` input, otherwise the `notebook_` with all the other views are visible.
     */
    ChannelView *searchView_{};
```

Correct the comment if it's wrong please!